ko-search/movies-evan-likes/src/App/components/Filters/index.js

```diff
@@ -1,8 +1,11 @@
-import React from 'react'
-import styles from './styles.css'
+import React, { Component } from 'react';
+import { centeredContainer } from 'sharedStyles.css';
+import { filter } from './styles.css';
 
-const Filters = () => (
-  <div className={styles.filters} />
-)
+const Filters = props =>
+  <div className={centeredContainer}>
+    <input className={filter} placeholder={'filter by title'} onChange={props.filterByTitle} />
+    <input className={filter} placeholder={'filter by decade'} onChange={props.filterByDecade} />
```

Comment:
You've pointed out a flaw with my Readme. When I said

> Allow searching by decade. Decade options should be computed from the data file, not hard-coded. Selecting a decade should filter the list to show all movies from that decade.

I really meant that to be a drop-down select menu, not a free-form entry. But I didn't explicitly say that and my screenshots are broken, so mea culpa.